clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -170,3 +218,19 @@ steps:
     Method: ServiceCreationLatency
     Params:
       action: gather
+{{if $NODEPOOL_UPGRADE_COMMAND}}
```

Comment:
Correct me if I'm wrong, but looking at implementation of ServiceCreationLatency I see that previously created measurements (for example "ServiceCreationLatencyLarge") will be collecting data throughout whole test. "waitForReady" doesn't seem to stop collecting data from informer so it will also take into account events during node pool upgrade?

Reply:
I did a little bit of research how ServiceCreationLatency works, and latency is always computed based on timestamp of service creation. Also, once measurement is started before node pool upgrade, service is reachable, so also reachability time might be incorrect too.